cloudscale/resource_cloudscale_objects_user.go

```diff
@@ -0,0 +1,150 @@
+package cloudscale
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/cloudscale-ch/cloudscale-go-sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceCloudScaleObjectsUser() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceObjectsUserCreate,
+		Read:   resourceObjectsUserRead,
+		Update: resourceObjectsUserUpdate,
+		Delete: resourceObjectsUserDelete,
+
+		Schema: getObjectsUserSchema(),
+	}
+}
+
+func getObjectsUserSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		// Required attributes
+
+		"display_name": {
+			Type:     schema.TypeString,
+			Required: true,
+		},
+
+		// Computed attributes
+
+		"href": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"user_id": {
```

Comment:
Not sure if this is possible, but everywhere else we call it `id`. It's probably just not possible, because the ID is already used by terraform. (You might have tried this even)

We should probably invest a bit of time to find out if it even makes sense to duplicate this here, because the information is available under `d.Id()`, but I'm not sure if the user can access this somehow. Just a thought. IMO it's not important, but if we can output `id` instead of `user_id` it would be nicer.

Reply:
I think it's unfortunately not possible.
https://github.com/hashicorp/terraform/commit/67ceb1ab07acd55ebc0d7b6efb39019e73fd1888